### DISCUSSION TITLE
Reduce size and center the plugin

### DISCUSSION THREAD

**Discussion by osdan-madrid** [iAlexeyProkhorov/qBoSlider#16] [General]
Hi!

I have been using your plugin and I find it very nice, thanks for developing it!
I have a question, in the site that I am developing, I installed it and it looks very good, but I would like to know if it is possible to do the following

I changed the size here:

<img width="238" alt="image" src="https://user-images.githubusercontent.com/42551335/230852685-f4df52a5-0c7b-4dd9-bc31-21500bfd9abf.png">

and it turned out very well like this, it's how I would like it to look:

<img width="846" alt="image" src="https://user-images.githubusercontent.com/42551335/230852801-957906ce-37f8-4ab8-97a1-7967321d4e03.png">

but is there a way to make it look centered? thus?

![image](https://user-images.githubusercontent.com/42551335/230853099-82cb8208-9221-4497-bf35-b94005620471.png)

I tried centering the content of the div but everything is out of balance:

<img width="401" alt="image" src="https://user-images.githubusercontent.com/42551335/230853727-e77219bd-ace3-4dd8-a767-902854713909.png">

<img width="491" alt="image" src="https://user-images.githubusercontent.com/42551335/230853782-eed47dfc-cbc8-48b5-a67c-a0988250f070.png">

Thanks in advance!

**Comment by iAlexeyProkhorov** [comment]
Hi!
I know that this message is late, but may be it will help somebody else.

Fast fix for this issue is next:
Need to modify CSS class 'slider-container'.
And there next two rows:    
```CSS
margin-left: auto;
margin-right: auto;
```
After this small fix all sliders will displays at widget zone center.

**Comment by iAlexeyProkhorov** [comment]
Hi! [Here ](https://github.com/iAlexeyProkhorov/qBoSlider/releases/tag/1.4.4)you can download pre-release of plugin version 1.4.4 for nopCommerce 4.60, which allows to set slider alignment.

**Reply by RichJeffries** [reply]
Hi there!
I've just installed this pre-release to control positioning, but it seems to be stretching the image and cutting off the left and right when I set the image dimensions...
![Screenshot 2024-08-03 111931](https://github.com/user-attachments/assets/721bed43-c98f-40e0-8627-417296c9ab93)

**Reply by RichJeffries** [reply]
This image is max 1200px wide, so it's cropping rather than scaling
